pages/common/r.md

```diff
@@ -0,0 +1,15 @@
+# R
+
+> R language interpreter.
+
+- Call a R interactive shell (REPL):
```

Comment:
I'd prefer `an` instead of `a` in this instance, as `R` is a single character.

Reply:
Fixed!